src/Billing/Controllers/PayPalController.cs

```diff
@@ -48,6 +48,7 @@ public class PayPalController : Controller
         [HttpPost("ipn")]
         public async Task<IActionResult> PostIpn()
         {
+            _logger.LogWarning("PayPal webhook has been hit.");
```

Comment:
Do we need these logs here, especially as a `WARN` level log (seems like a `DEBUG` level or `INFO` at most).